site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -12,8 +12,11 @@ The client SDKs are required for a game server to work with Agones.
 
 The current supported SDKs are:
 
+- [Unreal Engine]({{< relref "unreal.md" >}})
 - [C++]({{< relref "cpp.md" >}})
-- [Go](https://godoc.org/agones.dev/agones/sdks/go)
+- [Node.js]({{< relref "nodejs.md" >}})
+- [Go]({{< relref "go.md" >}})
+- [Rust]({{< relref "rust.md" >}})
```

Comment:
How about the local.md?

Reply:
We have a section below this:

> There is also local development tooling for working against the SDK locally, without having to spin up an entire Kubernetes infrastructure.

That work?